README.md

```diff
@@ -7,7 +10,31 @@ The PHG is a trellis graph based representation of genic and intergenic regions
 
 [PHGv1](https://bitbucket.org/bucklerlab/practicalhaplotypegraph/wiki/Home) was [published in 2022](https://doi.org/10.1093/bioinformatics/btac410). It addressed many challenges related to aligning diverse genomes, efficient storage, and imputation across a pangenome. However, it depended on a custom relational database that necessitated unique formats, and database queries did not scale effectively with a large number of taxa and rare alleles. Moreover, after developing PHGs for six species, we identified significant opportunities to refine and streamline the platform for curation.
 
-# PHGv2 design
+## Quick Start
+> [!IMPORTANT]
+> PHG2 is still under active development, please regularly check back for updates!
+
+You can download PHG2 from [the releases](https://github.com/maize-genetics/phg_v2/releases) page, but we recommend installing through Conda:
+
+```
+conda install --channel conda-forge --channel maize-genetics phg2
+```
+
+We recommend creating a dedicated Conda environment for each of your bioinformatics projects, as opposed to installing directly into the base environment; please refer to Conda's documentation [on managing environments](https://conda.io/projects/conda/en/latest/user-guide/getting-started.html#managing-environments). To create an environment and install PHG2 in a single command, do as follows:
+
+```
+conda create --name my_env --channel conda-forge --channel maize-genetics phg2
+```
+
+If you've already installed PHG2, you can update to the newest version via:
+
+```
+conda update phg2
+```
```

Comment:
Based on these changes, the documentation indicates the user can only execute phg from the conda setup (the original ./phg commands have been removed).  

Did we decide Conda is an option, or did we decide conda is the only option for installing phg?

If the latter, the instructions in the PHGv2- Building and Loading have an inconsistency.  Is our program named "phg" or "phg2" ? (we need to make a collective decision)

It looks like we'll have 2 environments - one that contains the phg(2) executable, and one that is created by phg(2) to run agc, tiledbvcf, etc.  Is that correct?

Reply:
BTW, I'm not opposed to loading phg only via conda.  I just want to ensure our documentation has consistent examples and that we're all on the same page.

Reply:
> Did we decide Conda is an option, or did we decide conda is the only option for installing phg?

I don't think we decided anything, this is a question for @zrm22 or the group, same as the naming question.

> It looks like we'll have 2 environments - one that contains the phg(2) executable, and one that is created by phg(2) to run agc, tiledbvcf, etc. Is that correct?

There is only a single environment, `phgv2-conda`. The other environment containing the actual program (the Conda package) isn't really relevant as I wrote in my long comment above.

Reply:
I think for now I would prefer explaining both the conda install and tarball installation instructions.  

I agree that long term if the user installs through conda we should only have one environment(no more need for SetupEnvironment) with all the dependencies, but I think it gets tricky as every time we make a call to anchorwave, tileDB, bgzip or bcftools we need to wrap the command that ProcessBuilder executes within a conda run command where we need to specify the environment name.  

The way we have this setup its consistent as we give the environment a name.  If the user needs to create their environment and then just add the PHG conda env to it, we no longer have this control.  

To fix either we need to have Processbuilder first figure out what envs are there and try to determine which one is the PHG one(what happens if you have more than 1?) or we have the user submit the name as a param(this would need to basically be in every command).  We may be able to have the user create an Environment variable that gets picked up by Clikt automatically, but it is hard to say.